sources/__tests__/getRole.js

```diff
@@ -43,34 +50,45 @@ function createElementFactory(tagName, attributes) {
 // prettier-ignore
 const cases = [
 	["a element with a href", "link", createElementFactory("a", { href: "any" })],
-	["a element without a href", null, createElementFactory("a", {})],
+	["a element without a href", "generic", createElementFactory("a", {})],
 	["abbr", null, createElementFactory("abbr", {})],
-	["address", null, createElementFactory("address", {})],
+	["address", "group", createElementFactory("address", {})],
 	["area with a href", "link", createElementFactory("area", { href: "any" })],
-	["area without a href", null, createElementFactory("area", {})],
+	["area without a href", "generic", createElementFactory("area", {})],
 	["article", "article", createElementFactory("article", {})],
+	// WARNING: Only in certain context
 	["aside", "complementary", createElementFactory("aside", {})],
 	["audio", null, createElementFactory("audio", {})],
+	["b", "generic", createElementFactory("b", {})],
 	["base", null, createElementFactory("base", {})],
-	["blockquote", null, createElementFactory("blockquote", {})],
-	["body", null, createElementFactory("body", {})],
+	["bdi", "generic", createElementFactory("bdi", {})],
+	["bdo", "generic", createElementFactory("bdo", {})],
+	["blockquote", "blockquote", createElementFactory("blockquote", {})],
+	["body", "generic", createElementFactory("body", {})],
+	["br", null, createElementFactory("br", {})],
 	["button", "button", createElementFactory("button", {})],
 	["canvas", null, createElementFactory("canvas", {})],
-	["caption", null, createElementFactory("caption", {})],
+	["caption", "caption", createElementFactory("caption", {})],
+	["cite", null, createElementFactory("cite", {})],
+	["code", "code", createElementFactory("code", {})],
 	["col", null, createElementFactory("col", {})],
 	["colgroup", null, createElementFactory("colgroup", {})],
+	["data", "generic", createElementFactory("data", {})],
 	["datalist", "listbox", createElementFactory("datalist", {})],
+	// WARNING: html-aria and html-aam conflict on this role assignment
```

Comment:
Can you link the sections showing this conflict? Ideally we'd open an issue and link to that.